client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -95,38 +152,50 @@ export class CommentDetailAuthor extends Component {
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="mail" />
 						<span>
-							{ authorEmail }
+							{ authorEmail || <em>{ translate( 'No email address' ) }</em> }
 						</span>
 					</div>
 					<div className="comment-detail__author-more-element">
 						<Gridicon icon="link" />
-						<ExternalLink href={ authorUrl }>
-							<Emojify>
-								{ urlToDomainAndPath( authorUrl ) }
-							</Emojify>
-						</ExternalLink>
+						{ !! authorUrl &&
+							<ExternalLink href={ authorUrl }>
+								<Emojify>
+									{ urlToDomainAndPath( authorUrl ) }
+								</Emojify>
+							</ExternalLink>
+						}
+						{ ! authorUrl &&
+							<em>{ translate( 'No website' ) }</em>
```

Comment:
Hi! I've found a possible matching string that has already been translated **106** times:
`translate( 'Website' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).